source/jormungandr/jormungandr/instance.py

```diff
@@ -476,9 +479,6 @@ def init(self):
         return False
 
     def get_street_network_routing_matrix(self, origins, destinations, mode, max_duration_to_pt, request, **kwargs):
-        if mode not in self.street_network_services:
```

Comment:
hum, even if the `mode` is checked by the API, I think it would be nice not to crash here (if we add another mode  for exemple)